front/lib/api/agent_data_sources.ts

```diff
@@ -0,0 +1,51 @@
+import type { ConnectorProvider } from "@dust-tt/types";
+import { Sequelize } from "sequelize";
+
+import type { Authenticator } from "@app/lib/auth";
+import { AgentDataSourceConfiguration } from "@app/lib/models/assistant/actions/data_sources";
+import { DataSource } from "@app/lib/models/data_source";
+
+export type DataSourcesUsageByAgent = Record<number, number>;
```

Comment:
Should be Record<ModelId, number>?

Reply:
Hmmm keys are the `dataSource.id`, from I've seen `ModelId` is for `connectorId`?

Reply:
It's the type for all models internal IDs 👍

Reply:
Alrighty 👍